### PR TITLE
Switch to epoll

### PR DESCRIPTION
**Pull request by edwintorok**
One small commit on top of https://github.com/xapi-project/xen-api/pull/5910 that switches from select to epoll ~everywhere in xapi/xenopsd.

Draft because we need to merge that PR first, then have a PR to update feature/perf from master.

Will need to be rebased on https://github.com/xapi-project/xen-api/pull/5913 once the prerequisites for that PR get merged.

### CODE REVIEW

**Comment by edwintorok** [comment]
We could also make the change suggested in https://github.com/xapi-project/xen-api/pull/5910#discussion_r1718347230 to use proper Sets.

File: ocaml/libs/ezxenstore/core/dune [hunk 1]
```diff
@@ -9,5 +9,6 @@
     (re_export xenstore)
     (re_export xenstore_transport)
     threads.posix
+    xapi-stdext-unix
```
Comment (psafont) [inline ocaml/libs/ezxenstore/core/dune:12]: This needs to be added to `ezxenstore.opam.template` as well

File: ocaml/xapi-idl/lib/dune [hunk 2]
```diff
@@ -32,6 +32,7 @@
    xapi-open-uri
    xapi-stdext-pervasives
    xapi-stdext-threads
+   xapi-stdext-unix
```
Comment (psafont) [inline ocaml/xapi-idl/lib/dune:35]: This needs to be added to `xapi-idl.opam.template` as well

**Review by psafont — COMMENTED** [review]
Just a couple of small notes

File: ocaml/libs/ezxenstore/core/dune (hunk 1 above)
Comment (edwintorok) [reply ocaml/libs/ezxenstore/core/dune:12]: Added as a fixup, will squash once people had a chance to review the other commits.

**Review by mg12 — APPROVED** [review]

**Pull request merged**